utils/bottlerocket/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2020 Brian Norris <computersforpeace@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=bottlerocket
+PKG_VERSION:=0.04c
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.linuxha.com/bottlerocket
+PKG_HASH:=deb8fbf8856f87af15fa9883e07b1fddb2aa78f6b4c7d119ae4bd664dc1d19ae
+
+PKG_MAINTAINER:=Brian Norris <computersforpeace@gmail.com>
+PKG_LICENSE:=LGPL
```

Comment:
Please use SPDX tag.

Reply:
Ah, sorry. Seems LGPL-2.1 (not later), judging by the linked license, so using that.